src/core/components/AMInstallButton/index.js

```diff
@@ -263,7 +263,11 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
     });
 
     const buttonIsDisabled =
-      disabled === false ? hasAddonManager && status === UNKNOWN : disabled;
+      disabled === false
+        ? hasAddonManager &&
+          status === UNKNOWN &&
+          addon.type !== ADDON_TYPE_OPENSEARCH
```

Comment:
Is this because we don't get a status from `mozAddonManager` for opensearch extensions?

Reply:
Yes.

I don't think there is a bugzilla issue for that given that opensearch plugins are installed with `window.external.AddSearchProvider` (which is why the opensearch add-ons have no status)

Reply:
Ah, that's fine. I forgot that it's a completely separate API.